lib/reports/internal-deadlines/index.js

```diff
@@ -43,13 +43,19 @@ module.exports = ({ db, query: params }) => {
     const q = db.flow('cases')
       .select([
         'cases.*',
-        db.flow.raw('JSON_AGG(activity_log.*) as activity')
+        db.flow.raw('JSON_AGG(activity_log.* ORDER BY activity_log.created_at) as activity')
       ])
       .leftJoin('activity_log', 'cases.id', 'activity_log.case_id')
       .whereRaw(`cases.data->>'model' = 'project'`)
       .whereRaw(`cases.data->>'action' = 'grant'`)
       .where('cases.updated_at', '>', earliest)
       .where('cases.created_at', '<=', latest)
+      .where(orBuilder => {
+        let statuses = [...targetClearingStatuses, ...closedStatuses];
+        return orBuilder.where(withinDateBuilder => withinDateBuilder.where('activity_log.updated_at', '>', earliest).where('activity_log.created_at', '<=', latest))
```

Comment:
It's a bit longer-winded like this but anywhere we're using builders it's probably easier to follow if it's nested rather than all on one line:

```suggestion
        orBuilder.where(withinDateBuilder => {
          withinDateBuilder
            .where('activity_log.updated_at', '>', earliest)
            .where('activity_log.created_at', '<=', latest);
        })
```